gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/pooled_grpc_asyncio.py.j2

```diff
@@ -0,0 +1,378 @@
+{% extends '_base.py.j2' %}
+
+{% block content %}
+
+import asyncio
+import warnings
+from functools import partialmethod
+from functools import partial
+from typing import Awaitable, Callable, Dict, Optional, Sequence, Tuple, Union, List, Type
+
+from google.api_core import gapic_v1
+from google.api_core import grpc_helpers_async
+{% if service.has_lro %}
+from google.api_core import operations_v1
+{% endif %}
+from google.auth import credentials as ga_credentials   # type: ignore
+from google.auth.transport.grpc import SslCredentials  # type: ignore
+
+import grpc                        # type: ignore
+from grpc.experimental import aio  # type: ignore
+
+{% filter sort_lines %}
+{% for method in service.methods.values() %}
+{{ method.input.ident.python_import }}
+{{ method.output.ident.python_import }}
+{% endfor %}
+{% if opts.add_iam_methods or api.has_iam_mixin %}
+from google.iam.v1 import iam_policy_pb2  # type: ignore
+from google.iam.v1 import policy_pb2  # type: ignore
+{% endif %}
+{% if api.has_location_mixin %}
+from google.cloud.location import locations_pb2 # type: ignore
+{% endif %}
+{% if api.has_operations_mixin %}
+from google.longrunning import operations_pb2
+{% endif %}
+{% endfilter %}
+from .base import {{ service.name }}Transport, DEFAULT_CLIENT_INFO
+from .grpc_asyncio import {{ service.grpc_asyncio_transport_name }}
+
+class PooledMultiCallable():
+
+    def __init__(self, channel_pool:"PooledChannel", *args, **kwargs):
+        self._init_args = args
+        self._init_kwargs = kwargs
+        self.next_channel_fn = channel_pool.next_channel
+
+class PooledUnaryUnaryMultiCallable(PooledMultiCallable, aio.UnaryUnaryMultiCallable):
+    def __call__(self, *args, **kwargs) -> aio.UnaryUnaryCall:
+        return self.next_channel_fn().unary_unary(*self._init_args, **self._init_kwargs)(*args, **kwargs)
```

Comment:
todo: add caching dict for unary calls for each channel, instead of re-creating each time